packages/kit-headless/src/components/carousel/carousel.test.ts

```diff
@@ -486,6 +486,34 @@ test.describe('Props', () => {
       // TODO
     });
   });
+
```

Comment:
looks great! I wonder if we can do this with a touch device as well, since we have two separate "modes" of interaction. Mouse with JS and Touch with CSS.